src/config/i18n/es/messages.json

```diff
@@ -17,6 +17,7 @@
   "dashboard.navigation.tab.mentorships": "Mentores",
   "dashboard.navigation.tab.messages": "Mensajes",
   "dashboard.navigation.tab.settings": "Ajustes",
+  "footer.legal.conduct": "",
```

Comment:
Spanish translation 🇪🇸 : Código de conducta

Reply:
Thank you! 😄

Reply:
Your welcome 😄